CODEOWNERS

```diff
@@ -243,21 +243,21 @@
 #
 # - Issue label: Migration
 # - Primary owner(s): @deepakrkris
-# - Standby owner(s): @nabdelgadir
-/packages/booter-lb3app @deepakrkris @nabdelgadir
-/packages/cli/generators/import-lb3-models @deepakrkris @nabdelgadir
-/examples/lb3-application @deepakrkris @nabdelgadir
-/docs/site/migration @deepakrkris @nabdelgadir
+# - Standby owner(s): @hacksparrow
+/packages/booter-lb3app @deepakrkris @hacksparrow
+/packages/cli/generators/import-lb3-models @deepakrkris @hacksparrow
+/examples/lb3-application @deepakrkris @hacksparrow
+/docs/site/migration @deepakrkris @hacksparrow
 
 #
 # REST CRUD conventional API
 #
 # - Issue label: n/a
-# - Primary owner(s): @nabdelgadir
+# - Primary owner(s): @raymondfeng
 # - Standby owner(s): n/a
-/packages/model-api-builder @nabdelgadir
-/packages/rest-crud @nabdelgadir
-/examples/rest-crud @nabdelgadir
+/packages/model-api-builder @hacksparrow
+/packages/rest-crud @raymondfeng
+/examples/rest-crud @raymondfeng
```

Comment:
I see the first entry is using `hacksparrow` but the remaining two are using `raymondfeng`. We usually want to use the same set of people for all packages belonging to the same function area.

BTW I am happy to take primary ownership of these packages myself, if that helps.

Reply:
Sure. IIRC, @hacksparrow also worked in this area too. @hacksparrow, would you like to be the standby owner?